src/components/cards/storyCard.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import CreateStories from "../../utilities/createStories";
+
+const StoryCard = ({
+  blok: {
+    storyPicker,
+    isMinimal,
+    isBigText,
+    hideTab,
+    hideImage,
+    headingLevel,
+    cardImageFocus,
+  },
+  isDark,
+}) => (
+  <CreateStories
+    stories={storyPicker}
+    layout="card"
+    isBigText={isBigText}
```

Comment:
Usually, the linter complains to me about the short syntax here: 

```suggestion
    isBigText
```